### PR TITLE
Add new env variable for CMS backend repo

### PR DESCRIPTION
**Pull request by tomrndom**
ref: https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskProject&view=vertical&task=3293186&stafilter=NotComplete&fileview=grid

Signed-off-by: Tomás Castillo <tcastilloboireau@gmail.com>

### CODE REVIEW

File: src/@openeventkit/event-site/utils/envVariables.js
Comment (smarcet) [inline src/@openeventkit/event-site/utils/envVariables.js:1]: do we really need to shadow this entire file ?

File: src/@openeventkit/event-site/cms/config/index.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import defaultConfig from "@openeventkit/event-site/src/cms/config";
 
+import { getEnvVariable, CMS_BACKEND_REPO } from "@openeventkit/event-site/src/utils/envVariables";
+
 const config = { ...defaultConfig };
-// TODO: move to env var
-config.backend.repo = "fntechgit/fnevent-tier1-default-theme";
+config.backend.repo = getEnvVariable(CMS_BACKEND_REPO);
```
Comment (smarcet) [inline src/@openeventkit/event-site/cms/config/index.js]: can we just do process.env.GATSBY_CMS_BACKEND_REPO ?

**Review by smarcet — CHANGES_REQUESTED** [review]
@tomrndom please review comments, also please add a .env.template file
regards

**Review by smarcet — APPROVED** [review]
LGTM

**Review by gcutrini — APPROVED** [review]
LGTM

**Comment by gcutrini** [comment]
Closing, since CMS repo configuration through env variable is now natively supported in @openeventkit/event-site package.

**Pull request closed without merging**